### PR TITLE
Move all file type (as in mime type) checks to MimeTypeUtil

### PR DESCRIPTION
**Pull request by AndyScherzinger**
PR for #317 [Refactoring] Utilize MimetypeIconUtil implementation

@tobiasKaminsky please review, etc. etc. :) 
- I tested it locally and it works on my device so it should be save :)

### CODE REVIEW

**Comment by MorrisJobke** [comment]
> Drone CI is currently blocked for unknown reasons... cc @MorrisJobke

Sorry - it's currently just overloaded - I will look into proper scale out soon :)

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky Imho ready to be merged - please review :rocket:

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky merge?

File: src/com/owncloud/android/utils/MimeTypeUtil.java [hunk 1]
```diff
@@ -121,28 +123,95 @@ public static String getBestMimeTypeByFilename(String filename) {
      * @return 'True' if the mime type defines image
      */
     public static boolean isImage(String mimeType) {
-        return (mimeType.startsWith("image/") && !mimeType.contains("djvu"));
+        return (mimeType!=null && mimeType.toLowerCase().startsWith("image/") && !mimeType.toLowerCase().contains
```
Comment (tobiasKaminsky) [inline src/com/owncloud/android/utils/MimeTypeUtil.java]: space before and after !=

Comment (AndyScherzinger) [inline src/com/owncloud/android/utils/MimeTypeUtil.java]: fixed in commit https://github.com/nextcloud/android/pull/323/commits/8a565f85db93292fb6f7efd1789f2c62f7763512

**Comment by AndyScherzinger** [comment]
And merging :+1: 

:rocket:

**Comment by AndyScherzinger** [comment]
lgtm has hickups...agin :rage:

**Pull request merged**